t/22-index.t

```diff
@@ -1,35 +1,7 @@
-use Test2::V0 -target => {pkg => 'Bagger::Storage::Index'};
-use strict;
-use warnings;
+use Test2::V0 -target => { pkg => 'Bagger::Storage::Index',
+                           dt  => 'Bagger::Type::DateTime'};
 
-plan 14;
```

Comment:
The constructor now needs a db connection of time fields are not supplied.  This is used for checking settings.  So the number of tests prior to the db run has been reduced.